docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1813,6 +1813,25 @@ If the block-builder permanently missed consuming some portion of the partition,
 - If the `block-builder-scheduler.lookback-on-no-commit` does not cover the time when the issue started, set it long enough so that these new block-builders start back far enough to cover the missing data.
 - Investigate why the block-builder fails, while the ingesters, who consumed the same data, don't.
 
+#### MimirBlockBuilderDataSkipped
+
+This alert fires when the block-builder-scheduler has detected a gap in either committed jobs or planned jobs.
+
+How it **works**:
+
+- Block-builder-scheduler is in charge of both planning "jobs" for block-builders to consume, as well as advancing the per-partition commit when one of these jobs is completed. Each job has a start offset (inclusive) and an end offset (exclusive).
+- Block-builder-scheduler also verifies that neither planning nor committing jobs ever produces a gap. This alert fires when the scheduler detects one of these gaps.
+- Note this is a "should never happen" problem, and will only happen if there's a correctness bug.
```

Comment:
```suggestion
- Note that this problem only happens if there's a correctness bug.
```

Reply:
I'm not totally clear on the meaning of "correctness bug". Is this a term Mimir users would be familiar with?

Reply:
There are bugs that crash, bugs that cause high resource usage, [skip many other classes of bugs...] and then bugs that cause an algorithm to work incorrectly. This would be the latter.

Good point- here we can just call it a bug in the planning logic.